### PR TITLE
Add CrateDB support for kontrol as a Storage backend

### PR DESCRIPTION
**Pull request by pdxjohnny**

### CODE REVIEW

File: kontrol/crate.go [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+	// cleanInterval := 120 * time.Second // clean every 120 second
+	// go p.RunCleaner(cleanInterval, KeyTTL)
+
+	return c
+}
+
+// Get retrieves the Kites with the given query
+func (c *Crate) Get(query *protocol.KontrolQuery) (Kites, error) {
+	return nil, fmt.Errorf("Not Impmentented")
+	// // We will make a get request to etcd store with this key. So get a "etcd"
```
Comment (rjeczalik) [inline kontrol/crate.go]: This should kites from crate directly, design-wise storages should be self-contained without dependencies.

If we wanted to read part from one storage and part from another, we create another storage, that would compose those storages. Like io.Writer and io.Readers are composed.

**Comment by rjeczalik** [comment]
Is it ready for a review? Adding new storage sounds like a cool feature, but I see part of the code is commented out.

**Comment by pdxjohnny** [comment]
This is not ready for review.

**Comment by gokmen** [comment]
@rjeczalik my mistake, I was trying to assign it to you. sorry @pdxjohnny

**Comment by pdxjohnny** [comment]
all good it may not be readyfor a whiel I just wanted to make others aware I was working on it.

**Pull request closed without merging**